MATLAB/DFA/src/EDA/exper_data.m

```diff
@@ -0,0 +1,360 @@
+T_std_HCT116_NT = Tabl_std_HCT116(:, {'metabolite', 'HCT116_shNT_no_dox_1','HCT116_shNT_2d_dox_1', 'HCT116_shNT_4d_dox_1', 'HCT116_shNT_no_dox_media_1', 'HCT116_shNT_2d_dox_media_1', 'HCT116_shNT_4d_dox_media_1'});
+
+%create numeric arrays
+avg_HCT116_GOT1 = table2array(T_avg_HCT116_GOT1(:, 2:end));
+avg_HCT116_NT = table2array(T_avg_HCT116_NT(:, 2:end));
+
+std_HCT116_GOT1 = table2array(T_std_HCT116_GOT1(:, 2:end));
+std_HCT116_NT = table2array(T_std_HCT116_NT(:, 2:end));
+
+% CELL LINE: TU9802
+T_avg_TU8902_GOT1 = Tabl_average_TU8902(:, {'metabolite', 'TU8902_shGOT1_no_dox_1','TU8902_shGOT1_2d_dox_1', 'TU8902_shGOT1_4d_dox_1', 'TU8902_shGOT1_no_dox_media_1', 'TU8902_shGOT1_2d_dox_media_1', 'TU8902_shGOT1_4d_dox_media_1'});
+T_avg_TU8902_NT = Tabl_average_TU8902(:, {'metabolite', 'TU8902_shNT_no_dox_1','TU8902_shNT_2d_dox_1', 'TU8902_shNT_4d_dox_1', 'TU8902_shNT_no_dox_media_1', 'TU8902_shNT_2d_dox_media_1', 'TU8902_shNT_4d_dox_media_1'});
+
+T_std_TU8902_GOT1 = Tabl_std_TU8902(:, {'metabolite', 'TU8902_shGOT1_no_dox_1','TU8902_shGOT1_2d_dox_1', 'TU8902_shGOT1_4d_dox_1', 'TU8902_shGOT1_no_dox_media_1', 'TU8902_shGOT1_2d_dox_media_1', 'TU8902_shGOT1_4d_dox_media_1'});
+T_std_TU8902_NT = Tabl_std_TU8902(:, {'metabolite', 'TU8902_shNT_no_dox_1','TU8902_shNT_2d_dox_1', 'TU8902_shNT_4d_dox_1', 'TU8902_shNT_no_dox_media_1', 'TU8902_shNT_2d_dox_media_1', 'TU8902_shNT_4d_dox_media_1'});
+
+%create numeric arrays
+avg_TU8902_GOT1 = table2array(T_avg_TU8902_GOT1(:, 2:end));
+avg_TU8902_NT = table2array(T_avg_TU8902_NT(:, 2:end));
+
+std_TU8902_GOT1 = table2array(T_std_TU8902_GOT1(:, 2:end));
+std_TU8902_NT = table2array(T_std_TU8902_NT(:, 2:end));
+
+%% Create a corelation matrix and a following heatmap
+%Do this by first converting all the NaN values into 0 and then create a corelation matrix of the metabolites using an array of averages.  
+% create arrays from the tables above. 
+mean_metabolomic_array_TU8902 = table2array(Tabl_average_TU8902(:,2:end));
+std_metabolomic_array_TU8902 = table2array(Tabl_std_TU8902(:,2:end));
+
+mean_metabolomic_array_HCT116 = table2array(Tabl_average_HCT116(:,2:end));
+std_metabolomic_array_HCT116 = table2array(Tabl_std_HCT116(:,2:end));
+
+%Convert all the NaN values to 0
+mean_metabolomic_array_TU8902(isnan(mean_metabolomic_array_TU8902)) = 0;
+mean_metabolomic_array_HCT116(isnan(mean_metabolomic_array_HCT116)) = 0;
+
+%r matrix and its heatmap - for the 12 conditions
+averageCorrMat_TU8902 = corrcoef(mean_metabolomic_array_TU8902);
+figure (1);
+heatmap(all_conditions, all_conditions, averageCorrMat_TU8902);
+title 'Correlation Matrix of TU8902 Averages across 12 Conditions';
+
+averageCorrMat_HCT116 = corrcoef(mean_metabolomic_array_HCT116);
+figure (2);
+heatmap(all_conditions, all_conditions, averageCorrMat_HCT116);
+title 'Correlation Matrix of HCT116 mean values over 12 conditions';
+
+% create r matrix and heatmaps for the metabolomics
+corrmat = zeros(12);

```

Comment:
I solved the issue with this bit of code: `corr` computes Pearson Correlation coefficients (by default) using the rows of your matrix. Instead of having your matrices be 12 x 226, have your matrices be 226 x 12. This will compute the metabolite-wise correlation coefficients.

An example snippet is shown below:
```corrmat = corr(mean_metabolomic_array_HCT116', mean_metabolomic_array_HCT116')```